urwid/widget/container.py

```diff
@@ -10,6 +11,16 @@
     from .widget import Widget
 
 
+class _ContainerElementSizingFlag(enum.IntFlag):
```

Comment:
this is only for simplified internal calculation
put to the base class since `Pile` support for FIXED widgets is already WIP and `GridFlow` is planned.
Adding support to `Pile` will allow `LineBox` to support FIXED widgets.